openedx_tagging/core/tagging/import_export/import_plan.py

```diff
@@ -207,8 +207,12 @@ def execute(self, task: TagImportTask | None = None):
         if self.errors:
             return
         for action in self.actions:
+            # Avoid to save each log because is slow and costs a lot in memory
```

Comment:
nit: typo "it is slow"

Reply:
Updated: https://github.com/openedx/openedx-learning/pull/189/commits/2130c8f30acec230b8bb551f14296820a249ec47